### PR TITLE
Add Governance Principle

### PR DESCRIPTION
**Pull request by cheyang**
Signed-off-by: cheyang <cheyang@163.com>

### Ⅴ. Special notes for reviews

### CODE REVIEW

**Review by TrafalgarZZZ — APPROVED** [review]

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+- **Open**: Fluid is open source community. 
+- **Welcoming and respectful**: See [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+- **Transparent and accessible**: Work and collaboration should be done in public.
+- **Merit**: Ideas and contributions are accepted according to their technical merit
+  and alignment with project objectives, scope and design principles.
+
+## Project Maintainers
+
+* Classify GitHub issues and perform pull request reviews for other maintainers and the community.
```
Comment (apc999) [inline GOVERNANCE.md:15]: Curate github issues and review pull requests

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+* During GitHub issue classification, apply all applicable [labels](https://github.com/fluid-cloudnative/fluid/labels)
```
Comment (apc999) [inline GOVERNANCE.md:17]: Remove " During GitHub issue classification"

File: GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  to each new issue. Use your best judgement to apply labels, since they are extremely useful for follow-up of future issues.
```
Comment (apc999) [inline GOVERNANCE.md:18]: Try your best to label issues which are very helpful and set a good example for future issues.

File: GOVERNANCE.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+## Project Maintainers
+
+* Classify GitHub issues and perform pull request reviews for other maintainers and the community.
+
+* During GitHub issue classification, apply all applicable [labels](https://github.com/fluid-cloudnative/fluid/labels)
+  to each new issue. Use your best judgement to apply labels, since they are extremely useful for follow-up of future issues. 
+
+* Maintainers are expected to respond to assigned Pull Requests in a reasonable time frame.
+
+* Participate when called upon in the security release process. Note
```
Comment (apc999) [inline GOVERNANCE.md:22]: not sure what you want to express here. Can you rephrase?

File: GOVERNANCE.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+  that although this should be a rare occurrence, if a serious vulnerability is found, the process
+  may take up to several full days of work to implement.
+
+* In general continue to be willing to spend at least 20% of your time working on Fluid (1 day per week).
```
Comment (apc999) [inline GOVERNANCE.md:26]: Expect to spend 20% time ...

File: GOVERNANCE.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+## Process of becoming a maintainer
+
+* Talk to one of the existing project [maintainers](MAINTAINERS.md) to show your interest in becoming a
+  maintainer. Becoming a maintainer generally means that you are going to be spending substantial
+  time (>20%) on Fluid for the foreseeable future. 
+
+* We will expect you to start contributing increasingly complicated PRs, under the guidance of the existing maintainers.
```
Comment (apc999) [inline GOVERNANCE.md:34]: We will expect you to start contributing pull requests with increasing complexity ...

File: GOVERNANCE.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+* We may ask you to do some PRs from our backlog. As you gain experience with the code base and our standards, 
+  we will ask you to do code reviews for incoming PRs.
+
+* After a period of approximately 3 months of working together and making sure we see eye to eye, the existing
```
Comment (apc999) [inline GOVERNANCE.md:39]: what does "eye to eye" mean here?

File: GOVERNANCE.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+on the issue, but not involved in the conflict) to intercede and the final decision will be made.
+Decision making process should be transparent to adhere to the principles of Fluid project.
+
+## Code of Conduct
+
+The Fluid [Code of Conduct](CODE_OF_CONDUCT.md) is aligned with the CNCF Code of Conduct.
+
+## Credits
+
+Some contents in this documents have been borrowed from [libvineyard](https://github.com/alibaba/libvineyard/blob/main/GOVERNANCE.md)
```
Comment (apc999) [inline GOVERNANCE.md:69]: This document references the GOVERNANCE guide from xxx and yyy

File: GOVERNANCE.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+
+* The Fluid community will never forcefully remove a current Maintainer, unless a maintainer fails to meet 
+  the principles of Fluid community.
+
+## Decision making process
+
+Decisions are made based on consensus between maintainers. In extreme cases, a simple majority voting process is invoked 
+where each maintainer receives one vote.
+
+Proposals and ideas can either be submitted for agreement via a github issue or PR.
```
Comment (apc999) [inline GOVERNANCE.md:56]: PR -> pull request , let's not use abbreviation in formal docs

File: CODE_OF_CONDUCT.md [hunk 10]
```diff
@@ -0,0 +1,5 @@
+# Fluid Community Code of Conduct
+
+Fluid follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+In cases of abusive, harassing, or any unacceptable behaviors, please don't hesitate to contact the [maintainers](MAINTAINERS.md).
```
Comment (apc999) [inline CODE_OF_CONDUCT.md:5]: don't -> do not

File: GOVERNANCE.md [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+
+## Process of becoming a maintainer
+
+* Talk to one of the existing project [maintainers](MAINTAINERS.md) to show your interest in becoming a
+  maintainer. Becoming a maintainer generally means that you are going to be spending substantial
+  time (>20%) on Fluid for the foreseeable future. 
+
+* We will expect you to start contributing increasingly complicated PRs, under the guidance of the existing maintainers.
+
+* We may ask you to do some PRs from our backlog. As you gain experience with the code base and our standards,
```
Comment (apc999) [inline GOVERNANCE.md:36]: We recommend you to take pull requests from the backlog.
As one grow more experienced and familiar with the code base and standards,
we expect him/her to review more pull requests

**Review by apc999 — CHANGES_REQUESTED** [review]

**Comment by apc999** [comment]
language polish. Please take a look

File: MAINTAINERS.md [hunk 12]
```diff
@@ -0,0 +1,10 @@
+# The Fluid Maintainers
+
+This file lists the maintainers of the Fluid project. The responsibilities of maintainers are listed in the [GOVERNANCE.md](GOVERNANCE.md) file.
+
+## Project Maintainers
+| Name | GitHub ID | Affiliation |
+| ---- | --------- | ----------- |
+| [Rong Gu](mailto:gurongwalker@gmail.com) | [RongGu](https://github.com/RongGu) | Nanjing University |
```
Comment (RongGu) [inline MAINTAINERS.md:8]: My main email address is gurong@nju.edu.cn, thanks!

**Review by apc999 — APPROVED** [review]

**Pull request merged**